pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/cliflags_test.go

```diff
@@ -40,7 +40,10 @@ var _ = Describe("CLIFlags", func() {
 	DescribeTable("#CLIFlags",
 		func(kubernetesVersion string, criName extensionsv1alpha1.CRIName, image *imagevector.Image, cliFlags components.ConfigurableKubeletCLIFlags, matcher gomegatypes.GomegaMatcher) {
 			v := semver.MustParse(kubernetesVersion)
-			Expect(kubelet.CLIFlags(v, criName, image, cliFlags)).To(matcher)
+			nodeLabels := map[string]string{
+				"test": "foo",
```

Comment:
Nit: To increase accuracy, you could have at least two entries here. Otherwise, the sorting and for-loop is not tested.